### ISSUE TITLE
Improving the UI 

### ISSUE THREAD

**Issue by Rashmisingh-18** [yashikabhargava/WebX#39]
Want to add some changes in UI like improving navbar, adding hover and transition effects to cards, adding favicon and improving the footer.

**Comment by Rashmisingh-18** [comment]
Hey @yashikabhargava, please assign this issue to me under hacktoberfest 2022.

**Comment by yashikabhargava** [comment]
@Rashmisingh-18 Can you please provide more information about the changes that you are planning on doing like
- How are you going to improve the navbar and footer?
And favicon is already being added by someone else so please do'nt add that as it will cause conflicts.

**Comment by shashank-iter** [comment]
@yashikabhargava Hey Yashika, actually I saw your repo on twitter and was trying to improve UI, I added some gradients and Glassmorphism to the Cards. Would you like to have a look ?

**Comment by shashank-iter** [comment]
Sure just doing that.

**Comment by Rashmisingh-18** [comment]
Hey @yashikabhargava, I raised this issue, would be glad if you assign this to me.

**Comment by yashikabhargava** [comment]
Yes. @Rashmisingh-18 I'll be assigning it to you only but please provide more information about the changes you are planning on doing like-

How are you going to improve the navbar and footer?
And favicon has already been added by someone else so please don't add that.

**Comment by Rashmisingh-18** [comment]
![Uploading Navbar.png…]()
In the navbar, want to add transition effects, borders, etc, improve that 'Get Started button.
![Uploading Footer.png…]()
And in the footer, want to include some more info, like 'quick links', 'Services', and 'Social Media handles.
Also after 'contact us', there should be a section that shows 'why you should choose this website for better learning'.

**Comment by yashikabhargava** [comment]
Sounds good, assigning this issue to you.

**Comment by Rashmisingh-18** [comment]
Made changes in Navbar as:
![navbar_webX](https://user-images.githubusercontent.com/90052358/193870807-82f0d544-4a50-4c4e-85bb-ec2745cef2ae.png)
And in the footer as:
![footer_webX](https://user-images.githubusercontent.com/90052358/193870876-f8bdc8f6-85df-4431-bfc5-4508b2e43154.png)

Please reply if any other changes are required.

**Comment by yashikabhargava** [comment]
Looking great, There is just a minor typo in the spelling of 'and' in the footer, Please fix that and then go ahead and open a PR.

**Comment by centurion6x** [comment]
hey @Rashmisingh-18  is your nav-bar persistent?
